src/sentry/tsdb/snuba.py

```diff
@@ -123,8 +107,74 @@ class SnubaTSDB(BaseTSDB):
             "times_seen",
             [["outcome", "=", outcomes.Outcome.FILTERED]],
         ),
+        TSDBModel.project_total_received_ip_address: SnubaModelQuerySettings(
```

Comment:
Cannot find where this model (nor the following one) is incremented. Do you know where this is happening ?

Reply:
Yea! It happens through [here](https://github.com/getsentry/sentry/blob/master/src/sentry/utils/outcomes.py#L97). Which gets the map from [here](https://github.com/getsentry/sentry/blob/master/src/sentry/utils/data_filters.py#L32).

Thanks for checking!